### PR TITLE
fixed avatar and message box layering

### PR DESCRIPTION
**Pull request by agrani29**
Signed-off-by: Agrani <eng.agraniverma@gmail.com>

Notes: Fix a bug (https://github.com/vector-im/element-web/issues/21446) where earlier the message bar was hiding behind the avatar for very small screens.

Before changes: 
Screenshot 2022-03-29 at 22 28 49

After changes: 
Screenshot 2022-03-29 at 22 29 41

How to review the change: 
Go to any room, and minimise the window to a very small size in the conversations. When you will hover over any conversation or will click on the sender's name, you will see that the message bar was hiding behind the avatar due to which it was not possible to perform any action from that message bar. 

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

**Comment by andybalaam** [comment]
@agrani29 looks like you need to fix one whitespace error that was shown by the failed build above:

```
res/css/views/messages/_MessageActionBar.scss
32:17  Unexpected whitespace at end of line
```

File: res/css/views/messages/_MessageActionBar.scss [hunk 1]
```diff
@@ -29,7 +29,8 @@ limitations under the License.
     right: 8px;
     user-select: none;
     // Ensure the action bar appears above over things, like the read marker.
-    z-index: 1;
+    z-index: 10;
+    //Edited the z-index so that the action bar appears above avatar in small screens
```
Comment (turt2live) [inline res/css/views/messages/_MessageActionBar.scss:33]: This comment should be above the z-index change, alongside the other one. Suggested re-wording:

> Ensure the action bar appears above over things like the read marker and sender avatar (for small screens)

**Review by turt2live — COMMENTED** [review]
generally looks great - thank you!

**Comment by andybalaam** [comment]
@agrani29 there is a just a small change you need to make and this can be merged.

File: res/css/views/messages/_MessageActionBar.scss (hunk 1 above)
Comment (andybalaam) [reply res/css/views/messages/_MessageActionBar.scss]: Done in 766d74b73179c39b229288044b53762c56e473a9

**Review by andybalaam — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**